src/ui/settings/ViewTipsAndTricks.qml

```diff
@@ -0,0 +1,130 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+
+import Mozilla.VPN 1.0
+import components 0.1
+
+Item {
+    id: root
+
+    property string _menuTitle: VPNl18n.SettingsTipsAndTricks
+    readonly property int tabletSize: 600 //Tablets get a different layout (2 columns for tutorials, 3 columns for guides)
+
+    objectName: "settingsTipsAndTricks"
+
+    VPNFlickable {
+        id: vpnFlickable
+
+        property int screenWidth: width
+
+        anchors.fill: parent
+        anchors.topMargin: VPNTheme.theme.menuHeight
+
+        flickContentHeight: layout.implicitHeight + layout.anchors.topMargin
+        interactive: flickContentHeight > height
+
+        ColumnLayout {
+            id: layout
+
+            anchors.top: parent.top
+            anchors.left: parent.left
+            anchors.right: parent.right
+            anchors.topMargin: VPNTheme.theme.vSpacing
+            anchors.leftMargin: VPNTheme.theme.windowMargin
+            anchors.rightMargin: VPNTheme.theme.windowMargin
+
+            spacing: 0
+
+            Flow {
+                Layout.fillWidth: true
+                spacing: VPNTheme.theme.vSpacingSmall
+
+                VPNTutorialCard {
+                    width: vpnFlickable.screenWidth < root.tabletSize ? parent.width : (parent.width - parent.spacing) / 2
+                    height: 144
+
+                    imageSrc: "qrc:/tutorials/01_get_started.svg"
```

Comment:
This is not good. You must use the model (or other c++ to qml layer). nothing has to be hardcoded for the following reasons:

1. Not all the tutorials can be exposed. See the `conditions` attribute in the JSON files. (Platforms, features, etc).
2. This must be dynamic. We cannot change the qml code each time we add/remove/change a tutorial.
3. It breaks the abstraction layer of the tutorial component.

Let's see how to do it properly. I'm sure we can find an elegant solution between you, Lesley and me :)

@lesleyjanenorton do you have any good hints to improve this code?

Reply:
This was a part of an older commit from a few days ago and these were just "dummy" cards I used to start laying out the page. The cards do get their data from the model as follows: 

```
VPNTutorialCard {
    property variant tutorial: VPNTutorial.getTutorialById("01_get_started")

    width: vpnFlickable.screenWidth < root.tabletSize ? parent.width : (parent.width - parent.spacing) / 2
    height: 144

    imageSrc: tutorial.image
    imageBgColor: VPNTheme.theme.ink //  TODO: REPLACE WITH DATA FROM MODEL ONCE BACKGROUND COLOR IS ADDED TO SCHEMA
    title: VPNl18n[tutorial.titleId]
    description: VPNl18n[tutorial.subtitleId]
}
```
You'll notice that this is still somewhat hardcoded by the property getting the tutorial object by knowing the tutorialId. That is because this card component is **NOT** a part of a component that takes in a model and creates delegates (eg ListView, Repeater). The reason for this is because of the way the tips and tricks page designed, not all tutorial cards are laid out next to each other, but they are divided into sections

![tipsandtricks](https://user-images.githubusercontent.com/15353801/167258161-780cab7e-73c5-4b46-8ba3-a8c7f2b1dd5d.png)

Right now we have one tutorial at the top (not including whats new because that doesn't get data from the model so presumably should remain completely hardcoded), and one at the bottom, which means they really can't be a part of the same component that lays out a whole model. 

The only way we can use those objects that lay out models is if we filter our models or have multiple models only containing the indices that we need for designated locations in the UI. For example, we could have a "header" tutorial model for the cards laid out at the top, and an "other" tutorial model for the headers laid out at the bottom, and we could add this data to this "designated location" to the json schema, or set up the file structure in such a way so that tutorials get put in the correct model, but because the future of this page is determined by feedback and usage, the layout may change slightly or dramatically, so it may be too soon to start splitting the monolithic tutorial model into sub/filtered models based on where they go in the view.

All that being said, if we think that there might be a better way to do this, I am definitely open to suggestions/brainstorming 😄 

Good call on the conditions though, I was not aware of those as I haven't gotten to actually implementing the guides/tutorials - just the card components right now, but I shall add logic to check this before displaying the cards.

Reply:
I have pushed [another, more up-to-date commit](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/3431/commits/2b0a602c7caa201034987a4b17add803f515322e) to better represent the direction I am going in. Let me know your thoughts!

Reply:
Ok i see! 
I would do the following:
1. Let's talk with Peter to see if we can simplify the view. 
2. The challenge here is too have a totally dynamic view: it must be able to support 0, 1, many tutorials. Same thing for guides.
3. The condition can not be checked in the view because..  unsupported tutorials are not loaded at all. Same thing for the guides
4. If needed, we can pick the first element from the model. Then we can use a VPN filterable model to take all the others. Could this be a solution?

Reply:
Let's bring this up in the design meeting. I think my preference would be to push the What's New card to the top and keep all of the tutorial cards together in a single block, like the guide cards.